### PR TITLE
further changes as requested in pr 391

### PR DESCRIPTION
**Pull request by jstoebel**
implementing additional changes as requested by @freakboy3742 from pr391

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -323,3 +323,133 @@ def test_complex_num(self):
         """)
 
         self.assertEqual(expected_out, self.cleaner.cleanse(js_in, {}))
+
+    def test_high_percision_float(self):
```
Comment (freakboy3742) [inline tests/test_utils.py]: Spelling! :-)

File: tests/test_utils.py [hunk 2]
```diff
@@ -323,3 +323,133 @@ def test_complex_num(self):
...
+
+        self.assertEqual(expected_out, self.cleaner.cleanse(js_in, {}))
+
+    def test_custom(self):
+        js_in = adjust("""
+        'we are the knights'
+        """)
+
+        expected_out = adjust("""
+        'we are the knights who go NI'
```
Comment (freakboy3742) [inline tests/test_utils.py]: They don't just *go* NI, they *say* NI :-)

Comment (jstoebel) [inline tests/test_utils.py]: oh gosh, how embarrassing! _gives Python license back_

File: tests/test_utils.py (hunk 1 above)
Comment (jstoebel) [inline tests/test_utils.py]: apologies for this. You had requested this in an earlier PR and I missed it.

**Pull request merged**